pystac_monty/geocoding.py

```diff
@@ -504,6 +504,10 @@ def get_iso3_from_geometry(self, geometry: Dict[str, Any]) -> Optional[str]:
         raise NotImplementedError("Method not implemented")
 
     def get_geometry_from_iso3(self, iso3: str) -> Optional[Dict[str, Any]]:
+        if not self.gpkg_path and self.service_base_url:
```

Comment:
Why dont have the world boundaries doing that instead of going on internet?

Reply:
We are using https://github.com/IFRCGo/geocoding-service